.idea/compiler.xml

```diff
@@ -2,8 +2,10 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <bytecodeTargetLevel target="1.8">
-      <module name="fhir-helper_jvmMain" target="1.6" />
-      <module name="fhir-helper_jvmTest" target="1.6" />
+      <module name="hc-fhir-helper-sdk-kmp.fhir-helper.commonMain" target="1.6" />
+      <module name="hc-fhir-helper-sdk-kmp.fhir-helper.commonTest" target="1.6" />
+      <module name="hc-fhir-helper-sdk-kmp.fhir-helper.jvmMain" target="1.6" />
+      <module name="hc-fhir-helper-sdk-kmp.fhir-helper.jvmTest" target="1.6" />
     </bytecodeTargetLevel>
   </component>
-</project>
\ No newline at end of file
+</project>
```

Comment:
Just out of curiosity - why sometimes Androidstudio puts a LF there and sometimes not...

Reply:
I don't know, could be related to the version of Android Studio and the project configuration, if it should ensure linefeed at file end or not.